contrib/qubes/test/test_qubes_fwupdmgr.py

```diff
@@ -0,0 +1,1038 @@
+#!/usr/bin/python3
+#
+# The Qubes OS Project, http://www.qubes-os.org
+#
+# Copyright (C) 2021  Norbert Kamiński  <norbert.kaminski@3mdeb.com>
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
```

Comment:
Would it be possible to license this and everything else LGPL instead?  The rest of the project is so that would be preferable.

Reply:
I've changed the license to LGPL.